TestCases/turbulence_models/sa/rae2822/turb_SA_NEG_RAE2822.cfg

```diff
@@ -1,3 +1,19 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+%                                  WARNING                                     %
+%                      This regression test will diverge!                      %
+%									       %
+% This test case is only meant to trigger the negative part of the SA model,   %
+% to check the integrity of the commits. It is inadvisable to use it as an     %
+% starting point for tutorials.						       %
+% The rae2822 airfoil is a simple (fast) case. A high angle of attack is being %
```

Comment:
For some reason github `Files changed` tab shows me a messed up indentation ... but looking at the actual file the indentation is on point. Not sure why github does that :shrug:

Reply:
hmm ... in chromium that looks good :thinking:

Reply:
![image](https://user-images.githubusercontent.com/31306376/158774820-d879efca-901a-49dc-bc7d-a89dc83177b9.png)